web/src/components/FilingsCalendar.test.tsx

```diff
@@ -649,4 +649,58 @@ describe("<FilingsCalendar />", () => {
       expect(screen.queryByText(Config.dashboardDefaults.calendarGridViewButton)).not.toBeInTheDocument();
     });
   });
+
+  describe("calendar list view, view more functionality", () => {
+    function renderCalendarWithEntries(numberOfCalendarEntries: number) {
+      let dueDate: dayjs.Dayjs;
+      let annualReport: TaxFiling;
+      const operateReferences: Record<string, OperateReference> = {};
+      const filings: TaxFiling[] = [];
+
+      for (let i = 0; i < numberOfCalendarEntries; i++) {
+        dueDate = getCurrentDate().add(2, "months").add(i, "day");
+        annualReport = generateTaxFiling({
+          identifier: `annual-report-${i}`,
+          dueDate: dueDate.format(defaultDateFormat),
+        });
+        filings.push(annualReport);
+        operateReferences[`annual-report-${i}`] = generateOperateReference({});
+      }
+
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: randomElementFromArray(
+            OperatingPhases.filter((obj) => {
+              return obj.displayCalendarType === "FULL" && obj.displayCalendarToggleButton;
+            })
+          ).id,
+        }),
+        taxFilingData: generateTaxFilingData({ filings: filings }),
+        preferences: generatePreferences({ isCalendarFullView: false }),
+      });
+
+      renderFilingsCalendar(operateReferences, userData);
+    }
+
+    it("does not show the view more button if 5 or less events are in the calendar", () => {
+      renderCalendarWithEntries(3);
+      expect(screen.getAllByTestId("calendar-list-entry")).toHaveLength(3);
+      expect(screen.queryByText("View More")).not.toBeInTheDocument();
```

Comment:
same comment as in the code - with externalized content, we don't want to hardcode any labels into the tests, because if content changes the label, we wouldn't want the tests to start failing. So the tests should reference the externalized content also, same as the code